src/index.ts

```diff
@@ -25,6 +43,13 @@ declare global {
       'astra-select': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
       'astra-calendar': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
       'outerbase-table': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
+      'code-mirror': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> & {
```

Comment:
I think we should change the naming everywhere from `code-mirror` to `astra-editor` if at all possible!